website/docs/cdktf/release/upgrade-guide-v0-13.mdx

```diff
@@ -0,0 +1,260 @@
+---
+page_title: Upgrading to CDKTF Version 0.13
+description: >-
+  Introducing namespaces for generated providers
+---
+
+# Upgrading to CDK for Terraform Version 0.13
+
+0.13 includes performance improvements to generated providers. Instead of exporting a flat list of exports
+including all supported resources and data sources, we now export each construct and it's associated structures in their own namespace.
+Due to this, the way you import constructs from your CDKTF application will change.
```

Comment:
We should add the redirected link to the blogpost / interim issue containing the reasoning for this change in this paragraph.